recipes/cec2/baseline/exp/exp/si_unproc.csv

```diff
@@ -0,0 +1,151 @@
+scene,listener,haspi

```

Comment:
This file exp/exp/si_unproc.csv shouldn't be included, there is already a exp/si_unproc.csv (worrying though because the numbers are different)